conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
@sameo 

> So my PR here passes a second argument to runc that is ignored (Which is another spec violation).

Actually what you're doing is telling runC to start multiple containers with one command (the second container name is invalid and so runC ignores it -- though it should return an error IIRC).